pkg/detectors/gitlab/gitlab.go

```diff
@@ -18,9 +20,11 @@ type Scanner struct {
 }
 
 // Ensure the Scanner satisfies the interfaces at compile time.
-var _ detectors.Detector = (*Scanner)(nil)
-var _ detectors.EndpointCustomizer = (*Scanner)(nil)
-var _ detectors.Versioner = (*Scanner)(nil)
+var (
```

Comment:
Out of curiosity, do you prefer the current way of doing the compile time interface check, or
```
var _ interface {
	detectors.Detector
	detectors.EndpointCustomizer
	detectors.Versioner
} = (*Scanner)(nil)
```

I only ask, b/c we've seen all 3 forms in the codebase, and I have an urge to standardize them 🤣

Reply:
Honestly, I hate all of them. Implicit interfaces feel like a mistake. 

With that being said, `_ detectors.Versioner = (*Scanner)(nil)` seems to be the most common pattern, but your alternative seems to be the most concise way to write it.